docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,98 @@
+Policy for Unofficial and Outdated Projects
+===========================================
+
+This policy describes a process where we take actions against unmaintained and unofficial forks of project documentation.
+
+Rationale
+---------
+
+There is often the case where a set of documentation has been built,
+and it is kept online but grows outdated over time.
+There are many reasons this can happen,
+including forks, old official docs that are unmaintained, and many other situations.
+
+The problem with old, outdated docs is that users will find them in Google,
+and get confused to the validity of them.
+Projects will then get support requests from people who are using an old and incorrect documentation version.
+
+We have this policy to allow project maintainers to request the *delisting* of forks that are old and outdated.
+
+Specification
+-------------
+The use cases covered by this policy are:
+
+Outdated docs
+    Documentation that is not being actively maintained or updated.
+
+Unofficial forks
+    Forks of a project documentation that are not linked to from the project in any way.
+
+Implementation
+--------------
```

Comment:
I'd put "Unofficial projects" and "Unmaintained projects" sections under "Definitions" or similar and leave "Implementation" for the rest that explain "how it works" and "what's the process" to achieve it.

Reply:
This is now moved :+1: